RELEASE-MASTERS.md

```diff
@@ -1,10 +1,85 @@
-# Releasing Knative
+# Release Masters
+
+For each release cycle, we dedicate a team of two individuals, one from Eventing
+and one from Serving, to shepherd the release process. Participation is
+voluntary and based on good faith. We are only expected to participate during
+our local office hour.
+
+# Roster
+
+We seed this rotation with all approvers from all the Serving and Eventing
+workgroups, excluding productivity. If you are no longer active in Knative, or
+if you are contributing on personal capacity and do not have time to contribute
+in the rotation, feel free to send a PR to remove yourself.
+
+## Serving roster
+- dprotaso
+- julz
+- JRBANCEL
+- markusthoemmes
+- mattmoor
+- nak3
+- tcnghia
+- vagababov
+- yanweiguo
+- ZhiminXiang
+
+## Eventing roster
+- evankanderson
+- grantr
+- Harwayne
+- lionelvillard
+- matzew
+- n3wscott
+- nachocano
+- slinkydeveloper
+- vaikas
+
+## Schedule
+
+| Release | Release Date | Serving        | Eventing        |
+|---------|--------------|----------------|-----------------|
+| v0.17   | 2020-08-18   | yanweiguo      | Harwayne        |
+| v0.18   | 2020-09-29   | ZhiminXiang    | n3wscott        |
+| v0.19   | 2020-11-10   | julz           | nachocano       |
+| v0.20   | 2020-12-22   | nak3           | slinkydeveloper |
+| v0.21   | 2021-02-02   | mattmoor       | lionelvillard   |
+| v0.22   | 2021-03-16   | markusthoemmes | evankanderson   |
+| v0.23   | 2021-04-27   | tcnghia        | vaikas          |
+| v0.24   | 2021-06-08   | dprotaso       | matzew          |
+| v0.25   | 2021-07-20   | vagababov      | grantr          |
+| v0.26   | 2021-08-31   | JRBANCEL       | ...             |
+
+# Release instruction
 
 We release the components of Knative every 6 weeks. All of these components must
 be moved to the latest "release" of all shared dependencies prior to each
 release.
 
----
+## 5 weeks prior to the release
+
+###  Make sure you are in the release-masters OWNER_ALIASES
```

Comment:
We don't need an OWNER_ALIAS, we just need a team in peribolos that gives the appropriate repo/org permissions.

Reply:
ACK, will update here once I set up the Peribolos role.

Reply:
Please take a look at https://github.com/knative/community/pull/209 ?

Reply:
Updated to include updating Peribolos.